Source/Scene/Cesium3DTile.js

```diff
@@ -429,6 +431,21 @@ define([
             }
         },
 
+        /**
+         * Application specific metadata.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {Object}
+         * @readonly
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#specifying-extensions-and-application-specific-extras|Extras in the 3D Tiles specification.}
+         */
+        extras : {
+            get : function() {
+                return this._extras;
```

Comment:
Instead of storing `_extras` as a member variable, this could just be `return this._header.extras`